pkg/process/procutil/process_linux_test.go

```diff
@@ -1090,3 +1094,13 @@ func maySkipLocalTest(t *testing.T) {
 		t.Skip("flaky test in CI")
 	}
 }
+
+// resetNiceValues takes a group of processes and reset the "nice" values on them.
+// this is needed because the "nice" values are not extract from procfs but using system call,
+// so it might cause test flakiness if we don't reset the value
+func resetNiceValues(procs map[int32]*Process) map[int32]*Process {
+	for _, p := range procs {
+		p.Stats.Nice = 0
+	}
+	return procs
+}
```

Comment:
You can just do
```suggestion
func resetNiceValues(procs map[int32]*Process) {
	for _, p := range procs {
		p.Stats.Nice = 0
	}
}
```
see [playground](https://play.golang.org/p/8Rs-Ps5vyG3)